### PR TITLE
fix(relocate-dashboard): fix relocate-dashboard issue

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [x] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
[QA - 727](https://pyengine.atlassian.net/browse/QA-727)
- Create `cost-explorer-settings-store.ts`.
- Apply settings store.
  - Registered in `CostExplorerContatiner.vue`.
  - Apply `CostExplorerLNB.vue`, `RelocateModal`, `RelocateNotification`.
- Fix settings vuex store issue.

### CODE REVIEW

File: apps/web/src/services/cost-explorer/CostExplorerContainer.vue [hunk 1]
```diff
@@ -44,13 +48,27 @@ export default {
...
+        costExplorerSettingsStore.initState();
+        costExplorerSettingsStore.$onAction((action) => {
+            action.after(() => {
+                if (window) {
+                    const settings = LocalStorageAccessor.getItem(userId.value) ?? {};
+                    settings.costExplorer = action.store.$state;
+                    LocalStorageAccessor.setItem(userId.value, settings);
+                }
+            });
+        });
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/CostExplorerContainer.vue:64]: praise: 👍

File: apps/web/src/services/cost-explorer/CostExplorerLNB.vue [hunk 2]
```diff
@@ -205,24 +210,20 @@ const handleLearnMoreRelocateNotification = () => {
...
 onMounted(() => {
     // Relocate dashboard notification
-    const settings = LocalStorageAccessor.getItem(relocateNotificationState.userId);
-    if (!settings.costExplorer?.hideRelocateDashboardNotification) {
-        relocateNotificationState.isModalVisible = true;
-        relocateNotificationState.isShow = true;
-    }
+    const relocateDashboardStatus: RelocateDashboardStatus|undefined = costExplorerSettingsStore.getRelocateDashboardStatus;
+    relocateNotificationState.isBannerVisible = !relocateDashboardStatus?.hideBanner;
+    relocateNotificationState.isModalVisible = !relocateDashboardStatus?.hideModal;
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/CostExplorerLNB.vue:226]: praise: I think it's far better to read

File: apps/web/src/store/modules/settings/actions.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ export const initSettings: Action<SettingsState, any> = ({ commit, rootState }):
     try {
         const settings = LocalStorageAccessor.getItem(userId);
 
-        if (settings) {
+        if (settings?.global) {
```
Comment (sulmoJ) [inline apps/web/src/store/modules/settings/actions.ts:13]: 🤓 🤓 🤓

File: apps/web/src/services/cost-explorer/store/cost-explorer-settings-store.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    actions: {
+        setRelocateDashboardState(state: CostExplorerSettingsStore['relocateDashboardStatus']) {
+            this.relocateDashboardStatus = state;
+        },
+        initState() {
+            const localStorageItem = initServiceSettingsStore<CostExplorerSettingsStore>('costExplorer');
+            this.relocateDashboardStatus = localStorageItem?.relocateDashboardStatus;
+        },
+    },
+});
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/store/cost-explorer-settings-store.ts:29]: what a big boilerplate 😅

**Review by sulmoJ — APPROVED** [review]
LGTM 🤓

File: apps/web/src/store/modules/settings/actions.ts (hunk 3 above)
Comment (WANZARGEN) [inline apps/web/src/store/modules/settings/actions.ts:13]: praise: oh.. that was the cause..

**Review by WANZARGEN — APPROVED** [review]
LGTM~!

**Pull request merged**